python/tests/integration/arcticdb/version_store/test_symbol_list.py

```diff
@@ -345,3 +347,8 @@ def test_symbol_list_parallel_stress_with_delete(
     missing_symbols = expected_symbols - got_symbols
     for sym in missing_symbols:
         assert not lib.version_store.indexes_sorted(sym)
+
+
+def test_symbol_list_exception_and_printout(mock_s3_store_with_mock_storage_exception):  # moto is choosen just because it's easy to give storage error
+    with pytest.raises(InternalException, match="E_S3_RETRYABLE Retry-able error"):
+        mock_s3_store_with_mock_storage_exception.list_symbols()
```

Comment:
Is this definitely hitting your new error message for exponential backoff? Not obvious to me how exactly we're exhausting the retry attempts?

Reply:
Yes, it is, definitely.
https://github.com/man-group/ArcticDB/blob/86eeb7cd2169f0264e763a1016f255fd7766fbce/python/arcticdb/storage_fixtures/s3.py#L224
`moto` will always return `503, slow down`.